test/parallel/test-crypto-from-binary.js

```diff
@@ -19,7 +19,7 @@ var ucs2_control = 'a\u0000';
 
 // grow the strings to proper length
 while (ucs2_control.length <= EXTERN_APEX) {
-  ucs2_control += ucs2_control;
+  ucs2_control = ucs2_control.repeat(2);
```

Comment:
That will loop until the string is just under length EXTERN_APEX but I think what we want is for it to loop until it exceeds EXTERN_APEX.

To fix that, you could simply add 1 to the number that results from the division. But by looping, I preserve the power of two length as well. That may not be significant, but I don't like to mess with crypto, so I did it this way so it was exactly the same.